Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,585 @@
+
+> For the purpose of this tutorial, we can add these to the existing security group, but in production it is a better choice to have 
+> separate security groups for your game servers and to not expose RDP
+
+```
+    ingress {
+        description = "Game Server Traffic"
+        from_port   = 7777
+        to_port     = 7777
+        protocol    = "udp"
+        cidr_blocks = ["0.0.0.0/0"] 
+    }
+
+    ingress {
+        description = "RDP Traffic"
+        from_port   = 3389
+        to_port     = 3389
+        protocol    = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        description = "RDP UDP Traffic"
+        from_port   = 3389
+        to_port     = 3389
+        protocol    = "udp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+```
+
+Next plan and apply your infrastructure
+
+```
+terraform plan -var-file="vars.tfvar" 
+terraform apply -var-file="vars.tfvar"
+```
+
+Once Terraform finishes, you can go to the AWS EC2 Console, click your instance, and click "Connect to instance." Under the RDP tab, download the
+Remote Desktop File. Then click "Get Password" and paste in the private key from the keypair you created previously.
+
+At this point you can connect to the Windows server using RDP by opening the file and logging into the machine with the "Administrator" account and password
+from the AWS console.
+
+At this point you should download and prepare to run your game on the server. Instructions for the Lyra Demo are below.
+
+<procedure title="Lyra Game Server Installation" id="lyra_setup" collapsible="true">
+    <p>These instructions use the <a href="https://github.com/catenatools/catena-lyra-demo">catena-lyra-demo</a> server.</p>
+    <step>Download the server onto the machine. The download can be found <a href="https://catena-public-content.S3.amazonaws.com/WindowsServer.zip">here.</a></step>
+    <step>Extract the zip file.</step>
+    <step>Run the game server from command prompt: <pre>LyraServer.exe -networkversionoverride=1234 -BackendUrl=localhost:8085</pre></step>
+    <step>Now the game server will communicate with the backend through the sidecar.</step>
+</procedure>
+
+## Run the Server Sidecar
+
+The server sidecar enables Catena to be aware of your game server without requiring any changes to the code of the game server itself. It is additionally able to make requests on behalf of your game server
+if you so choose. The sidecar will augment connection details and can translate requests between different backends so your server only needs to implement one SDK interface. See [Sidecar](Sidecar.md) for more details.
+
+First download the Sidecar to the machine, the latest release can be found [here](https://github.com/CatenaTools/sidecar/releases/). Alternatively, clone and build the sidecar.
+
+In the same directory as the sidecar, add a `config.json.` There is an example in the  sidecar repo here: https://github.com/CatenaTools/sidecar/blob/main/config.json
```

Comment:
Being more specific about "the sidecar" here may be good - this should be adjacent to the compiled binary. (User may download/clone and build themselves.)

Reply:
Missing hyperlink